test/arb/acb_mat-test.jl

```diff
@@ -116,7 +115,9 @@ end
       @test size(t) == (2, 3)
    end
 
-   println("PASS")
+   t = similar(s, CC, 2, 3)
+   @test t isa acb_mat
+   @test size(t) == (2, 3)
```

Comment:
These 3 lines should not have been re-added, as the same test is included in the loop `for (R, M) in ring_to_mat`.  Same in other files, I will force-push on the branch to fix the rebase. But thanks anyway!